examples/creatives/extender.js

```diff
@@ -22,13 +22,14 @@ class Extender extends BaseSimidCreative {
     super.onGetMediaStateResolve(data);
     const mediaDuration = data['duration'];
     const params = {
-      'duration': mediaDuration + 5,
+      'duration': mediaDuration - 35,
```

Comment:
I think you want to revert this, since you were just doing this for testing.

Reply:
yes